readme.md

```diff
@@ -51,7 +56,7 @@ ___
 
 ## Prerequisites
 
-- [NodeJS (14.x.x)](https://nodejs.org/en/) 
+- [NodeJS (16.x.x)](https://nodejs.org/en/) 
```

Comment:
Shall we ask users to try node 18 directly as 16 was deprecated.

Reply:
> Shall we ask users to try node 18 directly as 16 was deprecated.

@lirenhe I tend to keep consistence with our depending libraries, unless we're using some feature in 18. 16 is the minimal for `Typespec` (see [here](https://microsoft.github.io/typespec/introduction/installation#requirements)). Actually, I've been using node 16 for quite a long time until last week. There is no problem in 16.

Reply:
@lirenhe Updated according to the latest minimal requirement of `typespec`: https://azure.github.io/typespec-azure/docs/release-notes/release-2023-11-07#dropped-support-for-node-version-16